client/app/components/navbar/navbar.service.js

```diff
@@ -0,0 +1,36 @@
+{
+  class Navbar {
+    static buildMenuTitle(title) {
+      const { firstHalf, secondHalf } = Navbar.splitTextInHalf(title);
```

Comment:
splitTextInHalf? what are you trying to achieve here?
i don't understand how it could work in our multi-language context

Reply:
Good remark, I forgot to have this work on for FR :)